### PR TITLE
[BD-6] Use PyPi django-social-app

### PR DESCRIPTION
**Pull request by ericfab179**
Move social-app-django from github to base requirement, since PyPi version is now compatible with the platform.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ericfab179! I've created [BLENDED-357](https://openedx.atlassian.net/browse/BLENDED-357) to keep track of it in Jira.   More details are on the [BD-06](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1594000004/BD-06+Python+3.8+Upgrade) project page.


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: requirements/constraints.txt [hunk 1]
```diff
@@ -19,9 +19,6 @@ django-storages<1.9
 # Not properly supporting latest version of drf-jwt
 drf-jwt<1.15.0
 
-# the upgrade to 3.3.0 is what triggered users to lose superuser status; see ARCHBOM-1078 for details.
```
Comment (morenol) [inline requirements/constraints.txt:22]: @ericfab179  Did you check that problem?

**Comment by awais786** [comment]
No need for this PR now.

**Comment by natabene** [comment]
@awais786 Can you close please? I don't have closing rights.

**Comment by DawoudSheraz** [comment]
Closing this old PR.

**Comment by openedx-webhooks** [comment]
@ericfab179 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/course-discovery&entry.867055334=https://github.com/openedx/course-discovery/pull/2628&entry.1484655318=https://github.com/ericfab179&entry.752974735=2020-06-04+00:01&entry.1917517419=2022-08-22+11:22&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**